### ISSUE TITLE
Another game another hang.

### ISSUE THREAD

**Issue by KyonV2** [LoneGazebo/Community-Patch-DLL#2561]
1. Newest modpack 9.19

This is the second time we are cannot finish our game. 

Just switching turn from ~268>269 is enough to totally hang a game, and eventually crash. 

I don't know which log file you require for it.

https://mega.nz/#F!i1xHGZRR!wfOffzg9fnMek74KkYfq8Q

We're both using EUI version with updated DLL. 

Barbs, ruins, research agreements, event system. GG/GA points from barbs all on.

**Comment by LoneGazebo** [comment]
If you are getting a crash, make sure your minidump file is in there.

**Comment by KyonV2** [comment]
I did include the minidump file in the mega folder.

**Comment by KyonV2** [comment]
So, we decided to delete EUI and surprisingly it didn't crash YET. Maybe our pcs are just too bad to handle this game.

**Comment by LoneGazebo** [comment]
EUI is very resource intensive.

**Comment by Iamblichos** [comment]
As you get deep into the game, using EUI becomes much more likely to have a crash. To guarantee the most stable experience, I recommend you use the non-EUI version. Happy to see you getting that far into the game on multiplayer.

**Comment by KyonV2** [comment]
It's kinda sad to not use EUI, I really like some of the features it has and I've played a lot of games with it. Honestly surprising that a game from 2010 can be this resource heavy, I don't have the newest PC but I can run most of the stuff on medium/high settings, but this one crashes, we finished 3 games with NQ mod and EUI and it was working fine. Firaxis and their spaghetti coding strikes again, I guess. Yesterday when we were trying to figure out what was the cause of the crashes it took us 8 hours to play 21 turns, that's kinda ridiculous.
